node/src/components/consensus/highway_core/highway/vertex.rs

```diff
@@ -38,6 +43,15 @@ impl<C: Context> Dependency<C> {
     pub(crate) fn is_unit(&self) -> bool {
         matches!(self, Dependency::Unit(_))
     }
+    /// Returns `true` if both refer to the same vertex, even if they differ in whether they
```

Comment:
(Unresolving, so I don't forget. :sweat_smile: )

Reply:
In the final version of the code it looks like the newline is there. I commented on a _commit_ originally and I don't understand why GH doesn't show _outdated_ tag here.

Reply:
Ah, I see, thanks! Resolving again.
(It's not "outdated" because the comment is on the _next_ line, probably. Can't put a comment on a line that's not there.)